apps/gruenbuch/models.py

```diff
@@ -42,15 +43,15 @@ class GruenbuchOverviewPage(MetadataPageMixin, Page):
 
     page_intro_de = fields.RichTextField(
         blank=True, default="",
-        verbose_name="Page introduction de",
+        verbose_name=_('Page introduction de'),
         help_text=apps_blocks.HELPTEXT_RICHTEXT_A11Y)
```

Comment:
would it make sense to translate the constants too?

Reply:
oupps missed the blocks.py translations, nice one, will add :)